src/nvimage.js

```diff
@@ -564,3 +565,229 @@ NVImage.zerosLike = function (nvImage) {
   zeroClone.zeroImage();
   return zeroClone;
 };
+
+String.prototype.getBytes = function () {
+  let bytes = [];
+  for (var i = 0; i < this.length; i++) {
+    bytes.push(this.charCodeAt(i));
+  }
+
+  return bytes;
+};
+
+/**
+ * Returns nifti-1 header as ArrayBuffer.
+ * @param {nifti.NIFTI1} header nifti-1 header to convert
+ * @returns {ArrayBuffer}
+ */
+NVImage.convertNiftiOneHeaderToArrayBuffer = function (header) {
```

Comment:
Do we need to do this? What is the purpose of this function? Our WASM code only needs a few attributes, not the full header. I think this is left over from the old code... might be worth keeping this for some eventual data storage to disk (e.g. convert NIfTI1 to NIfTI2 and the reverse). However, I would remove vestigial code from the main product.

Reply:
I have left it in, because @cdrake put some work into this, but I have removed the methods from appearing in the public dev docs webpage

Reply:
This can be removed.  These methods are now in nifti-reader-js and was used in an older version of the WASM.